test/ethereumHelper.test.ts

```diff
@@ -0,0 +1,21 @@
+import { signEvmMessage } from "../src/common/helpers/ethereumHelper";
```

Comment:
Let the test be directly in the dedicated test file for everything chain 
`wallet.ethereum.test.ts`
`wallet.solana.test.ts`